frontend/src/reducers/usersReducer.tsx

```diff
@@ -20,6 +20,17 @@ const usersReducer = (state = initialState, action: IStoreUsersAction) => {
           items: [...action.pagedUsers.items],
         },
       };
+    case actionTypes.UPDATE_USER:
+      const items = [
+        ...state.pagedUsers.items.map((u: IUser) => (u.id === action.user.id ? action.user : u)),
```

Comment:
I'm curious, why is this necessary?

Reply:
What updating the user account status from the table, on success you also have to update/replace the object in the list